pandas/core/indexes/datetimelike.py

```diff
@@ -121,6 +121,45 @@ def floor(self, freq):
 
     @Appender(_round_doc % "ceil")
     def ceil(self, freq):
+        """
+        Floor the index to the specified freq.
+
+        Links to the available [frequency strings](http://pandas.
+        pydata.org/pandas-docs/stable/timeseries.html#offset-aliases)
+        and [frequency objects](http://pandas.pydata.org/pandas-docs/
+        stable/timeseries.html#dateoffset-objects).
```

Comment:
Can you use this long summary to explain what flooring the index means? The links look great, but if you check the source code of that pages, you should find a reference before them, that you can use here with a `:ref:`. Sphinx will build the links automatically with it.

Let me know if you need help with this.

Reply:
yes i need a help regarding  this :ref: @datapythonista

Reply:
At the beginning of this document [1] you can see this sentence: "See the :ref:\`cookbook<cookbook.missing_data>\` for some advanced strategies."

When the html documentation is generated with sphinx, this will look into the documentation for a bookmark `.. _cookbook.missing_data:` and create a link to it. That has some advantages. For example, it'll link to the same version of the documentation you're using, and not always the `stable` as your original implementation.

So, you should look into the source code of those links what's the name of the bookmark, and create them in the same way. If I'm not wrong, using the `--single` parameter when building the documentation something fails to create the links. It'll take longer, but build the whole documentation if that's the case.